src/base/TypeUtil.ml

```diff
@@ -638,7 +638,7 @@ module TypeUtilities = struct
                   %s is given"
                  (as_string name)
                  (TUName.as_string adt.tname))
-            (get_rep name)
+            lc
```

Comment:
We have `lc` specified in all call sites, excluding a call from `JSONParser.ml`. But it doesn't use annotations at all, and its `get_rep name` is always a `dummy_loc` anyway.